includes/class-dispatcher.php

```diff
@@ -405,4 +412,37 @@ protected static function should_send_to_followers( $activity, $actor, $outbox_i
 		 */
 		return apply_filters( 'activitypub_send_activity_to_followers', $send, $activity, $actor->get__id(), $outbox_item );
 	}
+
+	/**
+	 * Add Inboxes of Relays.
+	 *
+	 * @param array    $inboxes  The list of Inboxes.
+	 * @param int      $actor_id The Actor-ID.
+	 * @param Activity $activity The ActivityPub Activity.
+	 *
+	 * @return array The filtered Inboxes.
+	 */
+	public static function add_inboxes_of_relays( $inboxes, $actor_id, $activity ) {
+		// Check if follower endpoint is set.
+		$cc = $activity->get_cc() ?? array();
+		$to = $activity->get_to() ?? array();
+
+		$audience = array_merge( $cc, $to );
+
+		// Check if activity is public.
+		if ( ! in_array( 'https://www.w3.org/ns/activitystreams#Public', $audience, true ) ) {
+			return $inboxes;
+		}
+
+		$relays = \get_option( 'activitypub_relays', array() );
+
+		if ( empty( $relays ) ) {
+			return $inboxes;
+		}
+
+		$inboxes = array_merge( $inboxes, $relays );
+		$inboxes = array_unique( $inboxes );
```

Comment:
I think we can avoid calling this here, the function unique's it after the filter.